osu.Game.Rulesets.Osu/Mods/OsuModNinja.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Rulesets.Osu.Objects.Drawables;
+using osu.Game.Rulesets.Osu.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModNinja : Mod, IApplicableToDrawableHitObjects
+    {
+        public override string Name => "Ninja";
+        public override string Acronym => "NJ";
+        public override IconUsage? Icon => FontAwesome.Solid.UserNinja;
+        public override ModType Type => ModType.Fun;
+        public override string Description => "Slice the circles!";
```

Comment:
this name and description seem misleading. when I think of what is the core mechanic in fruit ninja, I think of slicing. this mod doesn't *actually* add slicing; from what I can see it solely has the hit objects emulate the way fruits fly into the playfield. (they don't fly back out of the screen either, just hang in their final position.)

(also not sure of potential legal repercussions of naming this after that game.)